nf_core/pipelines/create/create.tcss

```diff
@@ -59,10 +59,16 @@ HorizontalScroll {
     margin-left: 15;
     margin-right: 25;
     margin-bottom: 1;
+    overflow-y: scroll;
```

Comment:
```suggestion
    overflow-y: hidden;
```

Reply:
The height is changed with the class [`displayed`](https://github.com/nf-core/tools/pull/2404/files/25c9fa1c92b593a7970a1366c87d01429204f0c0..610cbd901e1d9fd9ebb76f5ef754c1b3f13b6964#diff-ea62118e0da89d39db497460edb0b4a8e98178632dfb7fcbfe1a951ceceec04cR71), maybe I should add `overflow-y: scroll` there instead of here